plugins/usbdmx/README.md

```diff
@@ -20,6 +21,12 @@ This plugin supports various USB DMX devices including:
 The debug level for libusb, see http://libusb.sourceforge.net/api-1.0/  
 0 = No logging, 4 = Verbose debug.
 
+`enable_eurolite_mk2 = {false,true}`
+Whether to enable detection of the Eurolite USB-DMX512 PRO MK2.
+Default = false. This device is indistinguishable from other devices
```

Comment:
I wonder, given it's got the SET BAUD RATE command, if there is some way to query it and prove it's not a normal FTDI, but I suspect that would still upset some other FTDIs anyway.

Reply:
I think all FTDI chips support this baudrate. I think the MK2 *needs* that particular baudrate settings to communicate with the internal cpu, but other FTDI chips probably would say "sure" after asking them to change to that baudrate. I think my open dmx supports the set baudrate command. Auto-selection would be nice though, but I don't know enough about the Eurolite MK2 to know how to do that.

Reply:
Ah yes, apologies, I assumed it was a Mk2 specific command, the Internet suggests it is indeed just FTDI_SIO_SET_BAUD_RATE. It's probably worth a comment to that effect to clarify. I wonder if Wiresharking the USB comms on Windows with some official software might reveal some other parts of the protocol, but possibly not.

Reply:
There is already a comment about it being indistinguishable, what are you thinking of to add?